lms/product/product.py

```diff
@@ -47,8 +47,12 @@ class Settings:
     groups_enabled: bool = False
     """Is the course groups feature enabled"""
 
+    custom: Optional[dict] = None
+    """Other non-standard settings."""
+
     def __post_init__(self, product_settings):
         self.groups_enabled = product_settings.get("groups_enabled", False)
```

Comment:
Do we want to `pop` here to ensure `custom` is _only_ non-standard things? This would prevent us from accessing `groups_enabled` directly and making different assumptions about the default.